client/disputes/evidence/file-upload.tsx

```diff
@@ -42,24 +43,30 @@ export const FileUploadControl = ( props ) => {
 		>
 			<DropZoneProvider>
 				<DropZone
-					onFilesDrop={ ( files ) =>
+					onFilesDrop={ ( files: Array< File > ) =>
```

Comment:
Can we try using `FileList` instead of `Array< File >` here (will the dropping files there still work with the change)?

Reply:
Apparently not xD Ignore the previous comment.

Reply:
I have tried putting `FileList` originally, but it throws an error `Array` type is expected. :)